test/test_misc.py

```diff
@@ -94,4 +102,4 @@ def test_rich_items(self):
     def test_rich_kvitems(self):
         self._assert_entry_point(JSON_KVITEMS, self.parse, self.kvitems, 'docs.item')
 
-generate_backend_specific_tests(globals(), 'MainEntryPoints', '', MainEntryPoints)
\ No newline at end of file
+generate_backend_specific_tests(globals(), 'MainEntryPoints', '', MainEntryPoints)
```

Comment:
Let's not have this unnecessary change here, thanks

Reply:
I think I've managed that. Could you please add trailing newlines to all files, though? Some editors just don't let you save files without that.